packages/phone-number-privacy/src/database/wrappers/number-pairs.ts

```diff
@@ -0,0 +1,48 @@
+import { getDatabase } from '../database'
+import { NUMBER_PAIRS_COLUMN, NUMBER_PAIRS_TABLE } from '../models/numberPair'
+
+const knex = getDatabase()
```

Comment:
Don't create a knex var here, as it could get out of sync with whatever knex object the database file is exposing. Recommend just calling getDatabase() everywhere you need this